lib/jsonapi_errorable/validatable.rb

```diff
@@ -1,8 +1,12 @@
 module JsonapiErrorable
   module Validatable
-    def render_errors_for(record)
-      validation = Serializers::Validation.new \
-        record, deserialized_params.relationships
+    # @param relationships: nil [ Hash or FalseClass ] list of relationships whose errors should be serialized
+    #   Defaults to the deserialized data.relationships of Json:api Payload
+    # @param record [ ActiveModel ] Object that implements ActiveModel
+    def render_errors_for(record, relationships: {})
+      relationships || deserialized_params.relationships
```

Comment:
this would never hit the right side of `||` since this defaults to `{}`. Maybe line 6 should be `nil`, but you can still pass a custom hash?